helpers.py

```diff
@@ -25,7 +26,8 @@ def public_charge(charge: Charges):
         "custom_css": charge.custom_css,
     }
 
-    if charge.paid:
+    if charge.paid or charge.zeroconf:
```

Comment:
- `or charge.zeroconf` is not correct, this means the `completelink` will be shown if the charge is created as `zeroconf` not if a transaction has been seen in the mempool

Reply:
hmmm... i see your point! not really sure how to handle it. i think i did it because the frontend expects the webhook link and was not showing the button when paid.